File: app/src/main/java/podtest/jiki/se/podtest/start/EpisodeDetailsPresenter.java
package podtest.jiki.se.podtest.start;

import podtest.jiki.se.podtest.model.Episode;

public class EpisodeDetailsPresenter {
    private EpisodeContract.View view;

    public EpisodeDetailsPresenter(EpisodeContract.View view) {
        this.view = view;
    }

    public void update(Episode episode) {
        view.setFirstRow(episode.getTitle());
        view.setSecondRow(episode.getDescription());
    }
}

File: app/src/main/java/se/asapehrsson/podtest/EpisodeDetailsPresenter.kt
package se.asapehrsson.podtest

import se.asapehrsson.podtest.model.Episode

class EpisodeDetailsPresenter(private val view: EpisodeContract.View) {

    fun update(episode: Episode) {
        view.setFirstRow(episode.title)
        view.setSecondRow(episode.description)
    }
}
